src/components/AddItem.jsx

```diff
@@ -0,0 +1,33 @@
+import { collection, addDoc } from 'firebase/firestore';
+import { db } from '../lib/firebase';
+
+export default function AddItem(userToken) {
+  // Accept a prop called newToken that should be a string
+  // Utilize AddItem component to create new collection in FireStore db for that specific user.
+  // user token should populate in FireBase database with new collection
+  console.log('AddItem token received =>', userToken);
+  console.log('typeof:', typeof userToken);
+
+  // This is the last task ==>
+  // It worked before with "Shopping-List" as argument but not working anymore..
+  // Unable to create new collection in database with newToken as is >>>
```

Comment:
It is still not working with `userToken`?

Reply:
Instead of passing userToken through as a property we now have the userToken saved in state and is now accessed only through saved state. Additem now has access to userToken this way.

Reply:
Great! That makes sense!